packages/playground-ui/src/domains/workflows/workflow/workflow-condition-node.tsx

```diff
@@ -142,11 +153,26 @@ export function WorkflowConditionNode({ data }: NodeProps<ConditionNode>) {
                   <Fragment key={`${condition.ref?.path}-${index}`}>
                     {condition.ref?.step ? (
                       <div className="flex items-center gap-1">
-                        {index === 0 ? null : (
-                          <Badge icon={<Network className="text-[#ECB047]" />}>
-                            {condition.conj?.toLocaleUpperCase() || 'WHEN'}
-                          </Badge>
-                        )}
+                        {index === 0
+                          ? null
+                          : (() => {
```

Comment:
Nitpicky:

IIFE are kind of hard to read, maybe you could shift the variable assignement between the ".map" call and the "return" thing.